core/feed.ts

```diff
@@ -100,6 +100,16 @@ export async function changeFeed(
   feedId: string,
   changes: Partial<FeedValue>
 ): Promise<void> {
+  if (changes.reading) {
+    let posts = await loadPosts({ feedId })
+    await Promise.all(
+      posts.map(post =>
+        changePost(post.id, {
+          reading: changes.reading
```

Comment:
Post’s type is not only the result of feed type, but also a result of filters.

This code will override filters. We need to use them here and also re-run the code on filter’s changes too. Maybe it will be a good idea to move post’s type calc to some function.

https://github.com/hplush/slowreader/blob/main/core/refresh.ts#L109-L117